pages/ecosystem.vue

```diff
@@ -402,3 +416,17 @@ const filteredMembersSorted = computed<Member[]>(() => {
   }
 }
 </style>
+
+<style lang="scss">
+@use "~/assets/scss/carbon.scss";
+
+.ecosystem-page__join-section {
+  .join-with-cards__feature {
+    height: 15.5rem;
+
+    @include carbon.breakpoint-down(md) {
+      height: 14.5rem;
+    }
+  }
+}
```

Comment:
Why do we need to set a fix height here? Is it because of the space between the tier description and the project count? If so, I believe this should be handled with a margin or a flex/grid gap.

Reply:
These heights were set a while ago when they were a part of the `AdvocatesJoin` component, I'm just overriding them to make the project count fit, without cramming content. 

https://github.com/Qiskit/qiskit.org/pull/3443/files#diff-51ac0a67d015e16594a8062805ae88a7ef4741c62735c55e77c618823d262087L75-L90

If we want to refactor these cards, we could handle it in a separate PR?

Reply:
I think that overriding is not a good idea in this case, to be honest. It leads to more technical debt.

In my opinion, we should get rid of these height rules in the **pages/ecosystem.vue** file. Instead, we should add a top margin in the **components/Ui/UiCompactFeature.vue** for the `ui-compact-feature__copy` `div` containing the `detail` text, and then also change the `height` properties for `min-height` properties for instance in the **components/Ui/UiJoinWithCards.vue** file.

Reply:
> _I think that overriding is not a good idea in this case, to be honest. It leads to more technical debt._

(I agree, good point)



Addressed [here](https://github.com/Qiskit/qiskit.org/pull/3477/commits/4718ee992e5c939a4ae206cfe5df3dee6470396d)